aldryn_config.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+from aldryn_client import forms
+
+
+class Form(forms.BaseForm):
+
+    def to_settings(self, data, settings):
+        from functools import partial
+        from aldryn_addons.utils import boolean_ish, djsenv
+        from aldryn_django import storage
+
+        env = partial(djsenv, settings=settings)
+
+        # django-filer
+        settings['FILER_DEBUG'] = boolean_ish(env('FILER_DEBUG', settings['DEBUG']))
```

Comment:
Agreed. For this first stage I think it's fine as we've not provided these options before.